README.md

```diff
@@ -97,6 +97,7 @@ The supported languages are listed below, you can use any of the `lang Values` i
 | `rus-ru`  | `rus` | Russian | RUS |
 | `esp-es`  | `esp` | Spanish | ES |
 | `esp-mx`  | `esp` | Spanish (default) | MX |
+| `ces-cz`  | `ces` | Czech | CZ |
```

Comment:
Thank you so much, I think this is fine just doing `ces`, but to think a bit about

> Let me know if you are using ISO 639-2/T (ces) or 639-2/B (cze).

I think we could support both for the query param.


```suggestion
| `ces-cz` or `cze-cz` | `cze` (B) `ces` (T) | Czech | CZ |
```

> iso639-2 has no collisions/duplicates of usage for `ces` or `cze` 

As I had not yet thought about this as much recently and wanted to see how larger organizations made their standards. 

Large companies like `Apple` default to 
> For a complete list of ISO 639-1 and ISO 639-2 codes, see [ISO 639.2 Codes for the Representation of Names and Languages](http://www.loc.gov/standards/iso639-2/php/English_list.php)
-- [Apple](https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPInternational/LanguageandLocaleIDs/LanguageandLocaleIDs.html)


> Czech | Czech | tchèque | cze/ces | cs
-- [ISO 639.2 LOC](http://www.loc.gov/standards/iso639-2/php/English_list.php)



What do you think? :thinking:

Reply:
I haven't seen three-letters language codes in years, everything uses `cs`/`cs_CZ` format these days. If I had to pick one, it would be `ces`, but it's 50:50.

Out of curiosity, why did you choose 639-2 over 639-1? Two-letter codes are used much more frequently for localization, from my experience.